### DISCUSSION TITLE
Adding instrumentation support for Ratpack 2.x version

### DISCUSSION THREAD

**Discussion by NaveenRamu** [open-telemetry/opentelemetry-java-instrumentation#12960] [Q&A]
As suggested in the [Git issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12317), I have copied the 1.7 instrumentation module and aligned the imports to Ratpack 2.x.

Questions:
1. How will the changes be pushed to the opentelemetry-java-instrumentation repo, as I don't have permission to push them?
2. If I don't receive permission to push the changes, how can I use these changes and write manual instrumentation (since manual instrumentation requires SDK and API dependencies)?

**Comment by steverao** [comment]
Duplicate problem with https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12962
